src/main/java/com/kakao/cafe/domain/User.java

```diff
@@ -3,40 +3,40 @@
 import java.time.LocalDateTime;
 
 public class User {
-    private String email;
-    private String userId;
-    private String password;
-    private LocalDateTime createdAt;
-
-    public LocalDateTime getCreatedDate() {
-        return createdAt;
-    }
-
-    public void setCreatedDate(LocalDateTime createdAt) {
-        this.createdAt = createdAt;
-    }
-
-    public String getEmail() {
-        return email;
-    }
-
-    public void setEmail(String email) {
-        this.email = email;
-    }
-
-    public String getUserId() {
-        return userId;
-    }
-
-    public void setUserId(String userId) {
-        this.userId = userId;
-    }
-
-    public String getPassword() {
-        return password;
-    }
-
-    public void setPassword(String password) {
-        this.password = password;
-    }
+
+  private String email;
+  private String userId;
+  private String password;
+  private LocalDateTime createdAt;
+
+  public User(String email, String userId, String password) {
+    this.email = email;
+    this.userId = userId;
+    this.password = password;
+    this.createdAt = LocalDateTime.now();
+  }
+
+  public LocalDateTime getCreatedAt() {
+    return createdAt;
+  }
+
+  public String getEmail() {
+    return email;
+  }
+
+  public String getUserId() {
+    return userId;
+  }
+
+  public String getPassword() {
+    return password;
+  }
+
+  public boolean isSameEmail(String email) {
```

Comment:
```suggestion
  public boolean isSameEmail(User other) {  
```
어떤 게 좋은 거 같아요? 가끔 고민하는데 경우에 따라 다른 것 같아요. 한 번 쯤 고민해 보세요.

Reply:
좀 더 살펴보고 고민해보겠습니다!